bundles/org.openhab.binding.emotiva/src/main/java/org/openhab/binding/emotiva/internal/EmotivaProcessorHandler.java

```diff
@@ -440,15 +471,22 @@ private void handleNotificationUpdate(AbstractNotificationDTO answerDto) {
     }
 
     private void handleChannelUpdate(String emotivaSubscriptionName, String value, String visible, String status) {
-        logger.debug("Handling channel update for '{}' with value '{}'", emotivaSubscriptionName, value);
+        if ("keepAlive".equals(emotivaSubscriptionName)) {
+            logger.trace("Subscription property '{}' with raw value '{}' received, start processing",
+                    emotivaSubscriptionName, value);
+        } else {
+            logger.debug("Subscription property '{}' with raw value '{}' received, start processing",
```

Comment:
Both should be debug or trace. As raw values are listed, it might be better to go for trace
```suggestion
            logger.trace("Subscription property '{}' with raw value '{}' received, start processing",
```

Reply:
Changing this really defeats the purpose of the if/else. I was trying to have the keepAlive notifications only in the trace level, as this really spams down the logs if one is doing debugging. The keep alive comes every 7,5 seconds, while other notifications does does not come that often.

Reply:
ah obviously, i should have noticed. I would suggest to remove the logic and always log to trace. Yës it is loud, but that is just where the trace log is meant for.